src/main/groovy/life/qbic/datamodel/datasets/OxfordNanoporeMeasurement.groovy

```diff
@@ -158,6 +159,24 @@ final class OxfordNanoporeMeasurement {
         }
     }
 
+    /**
+     * This method aggregates only *unclassified* fast5 files and fastq files of an Oxford Nanopore
+     * measurement by sample code.
+     *
+     * The resulting data-structure follows this map schema:
+     *
+     * "QBiC sample code":
+     *      "fast5fail": DataFolder
+     *      "fast5pass": DataFolder
+     *      "fastqfail": DataFolder
+     *      "fastqpass": DataFolder
+     *
+     * @return nested Map with sample codes and data folders
+     */
+    Map<String, Map<String, DataFolder>> getUnclassifiedDataPerSample(ExperimentFolder experiment) {
```

Comment:
unclassified data should not be stored per sample (code), there is only going to be one set of unclassified data per experiment!